### PR TITLE
Buffer's k8s APIs 50% test

### PR DESCRIPTION
**Pull request by federicoweber**
### Purpose
To test new Buffer's k8s APIs with 50% of analyze users

### Notes
This also adds in the service tag for rpc errors logging, that is useful to get a better picture of errors distribution.

#### Staging Deployment

_This repo is CI/CD enabled and staging deployment should be available at:
https://<branch_name>.analyze.buffer.com :smile:_

### CODE REVIEW

File: packages/server/index.js [hunk 1]
```diff
@@ -135,7 +135,7 @@ function getQuasiRandomAPIAddress() {
     return process.env.API_ADDR;
   }
 
-  const distribution = [0, 0, 0, 0, 0, 0, 0, 0, 1, 1];
+  const distribution = [0, 0, 0, 0, 0, 1, 1, 1, 1, 1];
```
Comment (federicoweber) [inline packages/server/index.js]: keeping this strategy as I want to slowly increase to 100% users

**Comment by federicoweber** [comment]
Our tests are still off — sorry about that 😞 —, but all related test suits are running fine.

**Comment by tiggreen** [comment]
Great, thanks Fed!

**Pull request merged**